### DISCUSSION TITLE
Perché la fattura generata dall'interfaccia riporta una frase del tipo "Copia del documento elettronico disponibile nella Sua area riservata dell'Agenzia delle Enterate"?  Come posso cancellare questa frase

### DISCUSSION THREAD

**Discussion by anxiglog** [fattureincloud/api#289] [Q&A] [answered]
Perché la fattura generata dall'interfaccia riporta una frase del tipo "Copia del documento elettronico disponibile nella Sua area riservata dell'Agenzia delle Enterate"?  Come posso cancellare questa frase

**Comment by valmoz** [comment, marked as answer]
Buongiorno @anxiglog 
ha realizzato questi documenti tramite il sito web oppure utilizzando le API?

Grazie
Mauro

**Reply by anxiglog** [reply]
API

**Reply by anxiglog** [reply]
![最新发票](https://user-images.githubusercontent.com/39998273/223603877-506fc319-656d-433f-a823-35c98164745a.png)

**Reply by emajo** [reply]
Ciao @anxiglog, quella dicitura viene messa di default su tutte le fatture elettroniche, per andare a rimuoverla è necessario andare a creare un template custom.

Grazie,
Emanuele.

**Reply by anxiglog** [reply]
grazie
